.nvmrc

```diff
@@ -0,0 +1 @@
+10
```

Comment:
I don't think we want this in the repo. It's up to individual developers to state their preferences here, no?

Reply:
Also, .nvmrc [requires a newline](https://github.com/creationix/nvm/tree/v0.33.11#nvmrc).

Reply:
Sure it is. This file will just help the maintainers and the contributors to use the same node version via nvm. It is not a necessary file but I think it definitely helps to switch between node version easily.

Reply:
Yes, but what if I don't want to switch to 10? I might be using 8 or 11.

It seems to me that this file should be there for an individual developer, but in .gitignore so everyone can use their own. And at least we have the script now to warn us if we're using the wrong version? Are there many projects that have .nvmrc in their repository?